pkg/workflow_template.go

```diff
@@ -575,6 +651,23 @@ func (c *Client) ListWorkflowTemplateVersions(namespace, uid string) (workflowTe
 	return
 }
 
+// ListWorkflowTemplateVersionsDB returns all the WorkflowTemplateVersions from the database.
+// This function is a work-around for ListWorkflowTemplateVersions. Once that function is refactored,
+// this function should no longer be necessary and removed.
+// See: https://github.com/onepanelio/core/issues/436
+func (c *Client) ListWorkflowTemplateVersionsDB(namespace, uid string) (workflowTemplateVersions []*WorkflowTemplateVersion, err error) {
```

Comment:
@aleksandrmelnikov 

Let's omit the DB suffix here. Ideally we don't export DB specific methods, so this method might be doing more work than just getting DB contents.

Reply:
Agreed